lib/client.ts

```diff
@@ -281,12 +285,12 @@ export class Client {
     )
   }
 
-  private request(
+  private request<T = any>(
     method: method,
     url: string,
     endpoint: string,
-    data?: any
-  ): Promise<any> {
+    data?: { [key: string]: any }
```

Comment:
Minor improvement here. Since data is treated like an object in this method, we can enforce that here.